cli/run_parser.py

```diff
@@ -104,24 +112,21 @@ def _get_files_to_parse(
     "--files",
     "-f",
     help="Pass in a list of filenames to parse, relative to the input directory. Used "
-    "to optionally specify a "
-    "subset of files to parse.",
+    "to optionally specify a subset of files to parse.",
```

Comment:
it might be nicer to put these multi-line docstrings in brackets for easier reading...?

e.g.:
```python
    help=(
        "Pass in a list of filenames to parse, relative to the input directory. Used "
        "to optionally specify a subset of files to parse."
    ),
```